Feature/draw semantic frame

PTavse opened the pull request:
The display colors for semantic keypoints and their bounding boxes are changed.

paul-michalik commented on src/FrameDrawer.cc:
```diff
@@ -112,18 +112,18 @@ cv::Mat FrameDrawer::DrawFrame()
 					if(vCurrentKeys[i].class_id != -1)
 					{
-						if(vCurrentKeys[i].class_id ==255)
+						/*if(vCurrentKeys[i].class_id ==255)
```
Please remove inactive code

PTavse commented:
@paul-michalik  Could you please have a look if the changes are good to go?

paul-michalik commented on src/FrameDrawer.cc:
```diff
@@ -112,19 +112,9 @@ cv::Mat FrameDrawer::DrawFrame()
-
+						cv::rectangle(im,pt1,pt2,cv::Scalar(0,0,255));
+						cv::circle(im,vCurrentKeys[i].pt,2,cv::Scalar(0,0,255),-1);
+						gNewSemanticFeatureCount++;
```
What is this variable for? Is it really global or is it just the name? Please get rid of global mutable variables, don't use them!

PTavse replied:
Yes Paul, I have removed the global variable in the code I am going to push under the issue https://github.com/paul-michalik/ORB_SLAM2/issues/31. (Show info about current frame)

PTavse commented:
It draws the semantic keypoints and the bounding boxes in red color. Yes it is connected to issue #27

PTavse commented:
@paul-michalik , could you please review? I have rebased the code and changed the semantic keypoints' color. Also I have removed a code which was no longer needed.

paul-michalik commented:
Let's consider this done. Closing!
